src/components/one-time-donation/components/Steps.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react'
+import FirstStep from './FirstStep'
+import SecondStep from './SecondStep'
+import ThirdStep from './ThirdStep'
+import Success from './Success'
+import Unsuccess from './Unsuccess'
+import {
+  DonationBankInput,
+  DonationResponse,
+  DonationStep as StepType,
+  OneTimeDonation,
+} from '../../../gql/donations'
+import { FormikStep, FormikStepper } from './FormikStepper'
+import { useTranslation } from 'next-i18next'
+import { useRouter } from 'next/router'
+import { useMutation, UseQueryResult } from 'react-query'
+import { CampaignResponse } from 'gql/campaigns'
+import { useViewCampaign } from 'common/hooks/campaigns'
+import { useVaultsList } from 'common/hooks/vaults'
+import { useCreateBankDonation } from 'service/donation'
+import { AxiosError, AxiosResponse } from 'axios'
+import { ApiErrors, isAxiosError, matchValidator } from 'service/apiErrors'
+import { FormikHelpers } from 'formik'
+import { validateFirst, validateSecond, validateThird } from '../helpers/validation-schema'
+
+const initialValues: OneTimeDonation = {
+  message: '',
+  anonymous: false,
+  amount: 0,
+  anonymousDonation: true,
+  personsFirstName: '',
+  personsLastName: '',
+  personsEmail: '',
+  personsPhone: '',
+  payment: 'bank',
+}
+
+export default function DonationStepper() {
+  const [status, setStatus] = React.useState(false)
+  const { t } = useTranslation('one-time-donation')
+  const router = useRouter()
+  const slug = String(router.query.slug)
+  const { data }: UseQueryResult<{ campaign: CampaignResponse }> = useViewCampaign(slug as string)
+  const valts = useVaultsList().data
```

Comment:
The regular users does not have access to this endpoint, it's used only by administrators

You can find the campaign valult only by via the view campaign endpoint

If the data is not provided, the API should be changed